playbooks/add_ssh_keys.yml

```diff
@@ -1,9 +0,0 @@
----
```

Comment:
Wouldn't it be good if we could run this on its own?

Reply:
You can by specifying the `ssh_keys` tag.

Reply:
I think it's good that we're now separating things that should only be run once on the initial setup from things that would reasonably be re-provisioned. 

In theory you can just run the whole playbook for `provision.yml` at any time and only the things that need changing will be changed, but yes you can run individual parts of it using `--tags ssh_keys` for example.

Reply:
> In theory you can just run the whole playbook for provision.yml at any time and only the things that need changing will be changed